@xen-orchestra/lite/src/components/pool/dashboard/AlarmRow.vue

```diff
@@ -0,0 +1,139 @@
+      <div v-tooltip class="ellipsis">
+        <ArrowCollapseButton :is-collapsed="!isDescriptionCollapsedRef" />
+        {{ $t(`alarm-type.${alarm.type}`, { n: alarm.triggerLevel * 100 }) }}
+      </div>
+    </td>
+  </tr>
+
+  <tr v-if="!isDescriptionCollapsedRef && isMobile" class="collapsible-description">
+    <td class="collapsible-description__content" colspan="5">
+      {{ tempDescription }}
+    </td>
+  </tr>
+  <tr v-if="isMobile" class="alarm-details" colspan="5">
+    <div v-tooltip="new Date(parseDateTime(alarm.timestamp)).toLocaleString()" class="ellipsis time">
+      <RelativeTime :date="alarm.timestamp" />
+    </div>
+    <td class="level typo h7-semi-bold">{{ alarm.level * 100 }}%</td>
+    <td class="on">{{ $t('on-object', { object: alarm.cls }) }}</td>
+    <td class="object">
+      <ObjectLink :type="rawTypeToType(alarm.cls)" :uuid="alarm.obj_uuid" />
+    </td>
+  </tr>
+
+  <tr v-if="isDesktop" class="desktop" @click="handleCollapse">
+    <th class="table-header">
+      <ArrowCollapseButton :is-collapsed="!isDescriptionCollapsedRef" />
+      <div v-tooltip="new Date(parseDateTime(alarm.timestamp)).toLocaleString()" class="ellipsis time">
+        <RelativeTime :date="alarm.timestamp" />
+      </div>
+    </th>
+    <td class="description">
+      <div v-tooltip class="ellipsis">
+        {{ $t(`alarm-type.${alarm.type}`, { n: alarm.triggerLevel * 100 }) }}
+      </div>
+    </td>
+    <td class="level typo h7-semi-bold">{{ alarm.level * 100 }}%</td>
+    <td class="on">{{ $t('on-object', { object: alarm.cls }) }}</td>
+    <td class="object">
+      <ObjectLink :type="rawTypeToType(alarm.cls)" :uuid="alarm.obj_uuid" />
+    </td>
+  </tr>
+  <tr v-if="!isDescriptionCollapsedRef && isDesktop" class="collapsible-description">
+    <td class="collapsible-description__content" colspan="5">
+      {{ tempDescription }}
+    </td>
+  </tr>
+</template>
+
+<script lang="ts" setup generic="T extends RawObjectType">
+import ObjectLink from '@/components/ObjectLink.vue'
+import RelativeTime from '@/components/RelativeTime.vue'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import { parseDateTime } from '@/libs/utils'
+import type { RawObjectType } from '@/libs/xen-api/xen-api.types'
+import { rawTypeToType } from '@/libs/xen-api/xen-api.utils'
+import type { XenApiAlarm } from '@/types/xen-api'
+import ArrowCollapseButton from '@/components/pool/dashboard/ArrowCollapseButton.vue'
+import { useUiStore } from '@core/stores/ui.store'
+import { storeToRefs } from 'pinia'
+import { ref } from 'vue'
+
+defineProps<{
+  alarm: XenApiAlarm<T>
+}>()
+
+const uiStore = useUiStore()
+const { isMobile, isDesktop } = storeToRefs(uiStore)
+const tempDescription =
+  'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus. Suspendisse lectus tortor, dignissim sit amet, adipiscing nec, ultricies sed, dolor. Cras elementum ultrices diam. Maecenas ligula massa, varius a, semper congue, euismod non, mi.'
```

Comment:
```suggestion
const tempDescription =
  'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus. Suspendisse lectus tortor, dignissim sit amet, adipiscing nec, ultricies sed, dolor. Cras elementum ultrices diam. Maecenas ligula massa, varius a, semper congue, euismod non, mi.'
```

A temporary description should be used while development only. Else, everyone will see this _lorem ipsum_ in their production app.